model/src/main/kotlin/ScanResult.kt

```diff
@@ -52,31 +50,12 @@ data class ScanResult(
     fun filterByPath(path: String): ScanResult {
         if (path.isBlank()) return this
 
-        val rootLicenseMatcher = RootLicenseMatcher(LicenseFilenamePatterns.getInstance())
-        val applicableLicenseFiles = rootLicenseMatcher.getApplicableRootLicenseFindingsForDirectories(
-            licenseFindings = summary.licenseFindings,
-            directories = listOf(path)
-        ).values.flatten().mapTo(mutableSetOf()) { it.location.path }
-
-        fun TextLocation.matchesPath() = this.path.startsWith("$path/") || this.path in applicableLicenseFiles
-
         val newProvenance = provenance.copy(
             vcsInfo = provenance.vcsInfo?.copy(path = path),
             originalVcsInfo = provenance.originalVcsInfo?.copy(path = path)
         )
 
-        val licenseFindings = summary.licenseFindings.filter { it.location.matchesPath() }.toSortedSet()
-        val copyrightFindings = summary.copyrightFindings.filter { it.location.matchesPath() }.toSortedSet()
-        val fileCount = mutableSetOf<String>().also { set ->
-            licenseFindings.mapTo(set) { it.location.path }
-            copyrightFindings.mapTo(set) { it.location.path }
-        }.size
-
-        val summary = summary.copy(
-            fileCount = fileCount,
-            licenseFindings = licenseFindings,
-            copyrightFindings = copyrightFindings
-        )
+        val summary = summary.filterByPath(path)
```

Comment:
commit-msg: our -> out

Reply:
The typo is still there...

Reply:
I didn't re-request a review until now as I had only fixed it locally.